config/initializers/03_sidekiq.rb

```diff
@@ -12,7 +12,7 @@
 
     config.death_handlers << ->(job, ex) do
       if ex.is_a?(Pender::Exception::RetryLater)
-        ex = Pender::Exception::RetryLimitHit.new(ex)
+        ex = Pender::Exception::RetryLimitHit.new(ex, {job: job})
       end
       Sentry.capture_exception(ex)
```

Comment:
What changes is here. We need to be able to send additional information along with the Sentry notification. I don't know if `Sentry.capture_exception` does that. If not, I suggest replacing this line by:
```
PenderSentry.notify(ex, { job: job })
```